doc/source/conf.py

```diff
@@ -105,14 +105,14 @@ def __getattr__(cls, name):
 
 if not on_rtd:
     # TODO: fix this once RTD updates their intersphinx version
-    extensions += ['sphinx.ext.intersphinx']
+    extensions.append('sphinx.ext.intersphinx')
 
     # Intersphinx to get numpy targets
     intersphinx_mapping = {
-        'python': ('http://python.readthedocs.org/en/latest/', None),
-        'numpy': ('http://numpy.readthedocs.org/en/latest/', None),
+        'python': ('http://python.readthedocs.io/en/latest/', None),
+        'numpy': ('http://numpy.readthedocs.io/en/latest/', None),
```

Comment:
This is dead since several months, we should link the public one, same goes for scipy.